store/src/java/com/zimbra/cs/volume/VolumeCLI.java

```diff
@@ -276,14 +276,10 @@ private void addVolume() throws ParseException, SoapFaultException, IOException,
         if (Strings.isNullOrEmpty(name)) {
             throw new ParseException(A_NAME + MISSING_ATTRS);
         }
-        if (Strings.isNullOrEmpty(path)) {
```

Comment:
This path is optional only in the case of EXTERNAL volumes, it should be mandatory param for INTERNAL right?

Reply:
Yes, this is mandatory in INTERNAL block validation only. In the case of EXTERNAL, Path will be optional and we are not using it at all. The default path will work in an EXTERNAL case.